### DISCUSSION TITLE
Trouble with student id verification

### DISCUSSION THREAD

**Discussion by dJaroo5** [community/community#42955] [GitHub Education]
I have 2 issues:
1. My school id is double sided. One side has all of the information and the second one has an expiry date. I can only submit 1 photo that I can only take through through my browser and can't choose any photos from my gallery.
2. When I tried taking a photo through my phone's browser it only allowed me to use the selfie camera.

**Comment by jfullstackdev** [comment]
there are similar issues like that here on GitHub Community

other posts were responded by the GitHub Staff, though there is still no assurance,

you may want to try Support too 

https://support.github.com/contact/account

**Comment by PrarthnaLuthra** [comment]
You can create a collage of the front and back sides of your id and upload it via browsing the document. It will work!
